### DISCUSSION TITLE
Iit seems like the package may be updated?

### DISCUSSION THREAD

**Discussion by drmoc** [ImageMagick/ImageMagick#7187] [Help]
### ImageMagick version

7.1.1-29

### Operating system

MacOS

### Operating system, version and so on

mac m1 arm // macOS 14.4 // Darwin 23.4.0

### Description

```bash
❯ luarocks install magick --check-lua-versions                                                                              at  12:53:09

magick not found for Lua 5.4.
Checking if available for other Lua versions...
Checking for Lua 5.1...
Checking for Lua 5.2...
Checking for Lua 5.3...

Error: No results matching query were found for Lua 5.4.
magick supports only Lua 5.1 but not Lua 5.4.
```

### Steps to Reproduce

When trying to use the image.nvim plugin for neovim, Luarocks magick is one of the prerequisites.
I can even install it using the command '~luarocks --local --lua-version=5.1 --force install magick', but it didn't work.
Is there an expected update? Thanks.

**Comment by dlemstra** [comment]
We don't maintain luarocks so you should ask your question somewhere else.

**Comment by drmoc** [comment]
Rigth
